checkVersionInSamTemplate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+readonly versionLine=$(grep 'SemanticVersion' sam/template.yaml)
+
+if [[ $versionLine == *"$1"* ]]; then
```

Comment:
Can this work? `versionLine` contains `SemanticVersion`, but `$1` does not

Reply:
Yes. The `*` acts as wildcard. So It's a `contains` in unreadable bash

Reply:
```suggestion
if [[ $versionLine == *"$1"* ]]; then # the * acts as wildcard --> it's a contains
```